ChangeLog.md

```diff
@@ -8,6 +8,13 @@
 - The client needs node version at least 5.4.
 - Add `raw GetNextUpdateSequenceNumbers` subcommand.
 - Add node version to the output of `raw GetNodeInfo`.
+- Add current timeout duration, current round, current epoch and trigger block time to the output
+  of `raw GetConsensusInfo`.
+- Add current timeout duration, current round, current epoch and trigger block time to the output
+  of `consensus status` when they are present.
+- Add round and epoch to the output of `raw GetBlockInfo`.
+- Add round and epoch to the output of `block show` when they are present.
+- Print "Block time" instead of "Slot time" in the output of `block show`.
```

Comment:
What about election difficulty as part of birk parameters? That's also printed as part of `printQueryBirkParameters` and is no longer present.

Reply:
This was added, so resolving.